docker/Dockerfile.cpu

```diff
@@ -0,0 +1,30 @@
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+
```

Comment:
use
```
FROM python:3.10-slim

# Install system dependencies
RUN apt-get update

# Install PyTorch manually
RUN pip3 install torch torchvision torchaudio --index-url https://download.pytorch.org/whl/cpu
```

Reply:
I know its less installing with the cuda one but iam afraid it will try to tackle drivers that it has no access to